pytorch_lightning/core/lightning.py

```diff
@@ -1473,6 +1478,64 @@ def untoggle_optimizer(self, optimizer_idx: int):
         # save memory
         self._param_requires_grad_state = {}
 
+    def clip_gradients(self, optimizer: Optimizer, gradient_clip_val: Union[int, float], gradient_clip_algorithm: str):
+        """Handles Gradient Clipping internally.
```

Comment:
```suggestion
        """Handles gradient clipping internally.
```
the capitalization seems arbitrary here, or is it referred to as a name somewhere else too?

Reply:
na.. just looks better in docs.